cd: problems while publishing as test flight

Since the last two PRs I get the following error:

```
*** SESSION AUTH: Jenga directives: {
    jenga = 1;
}
*** Error: Error uploading 'build/NOICommunity.ipa'.
*** Error: Unable to upload archive. The file 'build/NOICommunity.ipa' cannot be found. (-43)
Error: Process completed with exit code 1.
```

Any idea how to solve this?

--- a/NOICommunity/Info.plist
+++ b/NOICommunity/Info.plist
@@ -15,7 +15,7 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>CFBundleName</string>
+	<string>NOICommunity</string>
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>

